src/main/java/training/subsystems/elbow/Elbow.java

```diff
@@ -51,7 +51,7 @@ public void goToAngle(Rotation2d targetAngle) {
 	}
 
 	public boolean isAtAngle(Rotation2d targetAngle) {
-		return Math.abs(targetAngle.getDegrees() - motor.getEncoder().getPosition()) == 0;
+		return Math.abs(targetAngle.getDegrees() - motor.getEncoder().getPosition()) <= ElbowConstants.TOLERANCE.getRadians();
```

Comment:
targetAngle.getDegrees is Degrees
 motor.getEncoder().getPosition() is Rotations (Also, you already wrote a getPosition() function, use it instead)
and ElbowConstants.TOLERANCE.getRadians() is radians
You should match all units, otherwise it will go kaboom
```suggestion
		return Math.abs(targetAngle.getDegrees() - getPosition().getDegrees()) <= ElbowConstants.TOLERANCE.getDegrees();
```
![image](https://github.com/user-attachments/assets/c3cdce11-9263-4f35-bc0d-5a64757e26af)